tests/unit/channel_tests.py

```diff
@@ -885,25 +912,25 @@ def test_queue_purge_rpc_request(self, rpc, _unused):
     def test_queue_purge_rpc_request_nowait(self, rpc, _unused):
         self.obj._set_state(self.obj.OPEN)
         mock_callback = mock.Mock()
-        self.obj.queue_purge(mock_callback, 'foo', nowait=True)
+        self.obj.queue_purge('foo', nowait=True, callback=None)
         rpc.assert_called_once_with(
-            spec.Queue.Purge(0, 'foo'), mock_callback, [])
+            spec.Queue.Purge(0, 'foo'), None, [])
 
     def test_queue_unbind_raises_channel_closed(self):
         self.assertRaises(exceptions.ChannelClosed, self.obj.queue_unbind,
                           None, 'foo', 'bar', 'baz')
```

Comment:
The None maps to the old callback arg and should be moved to the end and use kwarg